src/util/apiUtils.js

```diff
@@ -317,10 +320,22 @@ export function getAuthHeaders(request) {
 	};
 }
 
+export function getLanguageHeader(request) {
+	const cookieLang = getCookieLang(request);
+	const headerLang = request.headers['accept-language'];
+	const acceptLang = cookieLang ?
+		`${cookieLang},${headerLang}` :
+		headerLang;
+	return {
+		'accept-language': acceptLang,
+	};
+}
+
 export function parseRequestHeaders(request) {
 	const externalRequestHeaders = {
 		...request.headers,
```

Comment:
Hmmm I wonder why it wasn't getting passed from the original request as it is passing accept-language. 
![screen shot 2017-04-25 at 2 15 56 pm](https://cloud.githubusercontent.com/assets/1933722/25400797/b24e718c-29c1-11e7-864e-d5f88d38ca66.png)